packages/app/pages/index.tsx

```diff
@@ -1,13 +1,197 @@
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+        }}
+        onKeyDown={onContainerKeyDown}
+        tabIndex={-1}
+      >
+        <div
+          style={{
+            maxWidth: 600,
+            minWidth: 400,
+            margin: 32,
+            boxShadow: "0px 12px 50px -24px rgba(0, 0, 0, 0.24);",
+            background: "#FDF3E7", // sand
+          }}
+        >
+          <div
+            style={{
+              height: 56,
+              backgroundColor: "#324841", // forest
+              paddingTop: 12,
+              paddingLeft: 40,
+              paddingRight: 16,
+              paddingBottom: 12,
+              display: "flex",
+              justifyContent: "space-between",
+              alignItems: "center",
+            }}
+          >
+            <span
+              style={{
+                fontSize: 18,
+                fontWeight: 600,
+                fontFamily: "IBM Flex Mono",
+                color: "#FDF3E7", // sand
+              }}
+            >
+              {title}
+            </span>
+            <button
+              style={{
+                width: 32,
+                height: 32,
+                backgroundColor: "#FDC67B", // wheat
+              }}
+              onClick={close}
+            >
+              X {/*replace with icon*/}
+            </button>
+          </div>
+          <div style={{ padding: 40 }}>
+            {children}
+            <div style={{ display: "flex", justifyContent: "right" }}>
+              <button // Replace with TOPO button
+                onClick={close}
+              >
+                Cancel
+              </button>
+              <button // Replace with TOPO button
+                onClick={onConfirmClick}
+                style={{ marginLeft: 32 }}
+              >
+                {confirmText}
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </Portal>
+  ) : null;
+};
+
+const MembershipTabContent = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const open = useCallback(() => setIsOpen(true), [setIsOpen]);
+  return (
+    <>
+      <h1>Memberships</h1>
+      <div>
+        <button style={{ padding: 8 }} onClick={open}>
+          Create New Membership Type
+        </button>
+        <Modal
+          isOpen={isOpen}
+          setIsOpen={setIsOpen}
+          title="New Membership Type"
+        >
+          <div style={{ paddingLeft: 8, marginBottom: 32 }}>
+            <input
+              name={"Name"} // TODO Create INPUT component in TOPO
+            />
+          </div>
+          <div style={{ paddingLeft: 8, marginBottom: 32 }}>
+            <input name={"Description"} />
+          </div>
+        </Modal>
+      </div>
+    </>
+  );
+};
+
+const UsersTabContent = () => {
+  return (
+    <>
+      <h1>Users</h1>
+      <div>Coming Soon!</div>
+    </>
+  );
+};
+
+const SettingsTabContent = () => {
```

Comment:
These tabs I selected were somewhat arbitrary. I envision us getting more clarity on what they should actually be as the product evolves